pkg/apis/integreatly/v1alpha1/grafanadatasource_types.go

```diff
@@ -142,6 +142,8 @@ type GrafanaDataSourceJsonData struct {
 	// Fields for Loki data sources
 	MaxLines      int                                  `json:"maxLines,omitempty"`
 	DerivedFields []GrafanaDataSourceJsonDerivedFields `json:"derivedFields,omitempty"`
+	// Fields for Prometheus data sources
+	customQueryParameters string `json:"customQueryParameters,omitempty"`
```

Comment:
```suggestion
	CustomQueryParameters string `json:"customQueryParameters,omitempty"`
```
Minor change, the field needs to be exported, I couldn't get it to work with an unexported one, however exporting it makes it work as intended, LGTM otherwise :+1: